commands/upgrade/breadcrumbs.py

```diff
@@ -59,6 +110,8 @@ def save(self):
                 crumbs.truncate()
                 json.dump(doc, crumbs, indent=2, sort_keys=True)
                 crumbs.write('\n')
+                if os.environ.get('LEAPP_NO_RHSM_FACTS', '0') == '1':
+                    self._save_rhsm_facts(doc['activities'])
```

Comment:
It should be opposite. When defined "1", facts should not be shared. Now it means, that facts are shared when `LEAPP_NO_RHSM_FACTS=1`. 

Also we should introduce an option like `--no-rhsm-facts` and probably the use of `--no-rhsm` should involve skipping of the use of RHSM also as: "Use only custom repositories and skip actions with Red Hat Subscription Manager".

Reply:
Both concerns were addressed with the latest commit.